gm4_orbis/data/gm4_orbis/functions/structure/chest_tower.mcfunction

```diff
@@ -8,3 +8,22 @@ execute if score chest_height gm4_count matches 90..109 run data merge block ~ ~
 execute if score chest_height gm4_count matches 110..129 run data merge block ~ ~ ~ {LootTable:"gm4_orbis:chests/tower_110"}
 execute if score chest_height gm4_count matches 130..149 run data merge block ~ ~ ~ {LootTable:"gm4_orbis:chests/tower_130"}
 execute if score chest_height gm4_count matches 150.. run data merge block ~ ~ ~ {LootTable:"gm4_orbis:chests/tower_150"}
+
+# set the loot table seed
+data modify storage gm4_orbis:temp Pos set from entity @s Pos
+execute store result score $x gm4_orbis_config run data get storage gm4_orbis:temp Pos[0] 64
+execute store result score $y gm4_orbis_config run data get storage gm4_orbis:temp Pos[1] 64
+execute store result score $z gm4_orbis_config run data get storage gm4_orbis:temp Pos[2] 64
+data remove storage gm4_orbis:temp Pos
+
+scoreboard players operation $loot_table_seed gm4_orbis_config = #world_seed gm4_orbis_config
+scoreboard players operation $loot_table_seed gm4_orbis_config *= $x gm4_orbis_config
+scoreboard players operation $loot_table_seed gm4_orbis_config *= $y gm4_orbis_config
+scoreboard players operation $loot_table_seed gm4_orbis_config *= $z gm4_orbis_config
+
+execute store result block ~ ~ ~ LootTableSeed long 256 run scoreboard players get $loot_table_seed gm4_orbis_config
```

Comment:
What is the math behind this? Why is each coordinate multiplied my 64, then added all together? And why is the resulting seed always a multiple of 256?

Reply:
Good point, I did it cuz I was thinking that they'd need to be multiplied by something anyways for the decimals, but those are always aligned, and for the 256 I wanted nearby chests to get seeds that were farther from each other, but I guess it doesn't matter as long as they're all different